pages/k8s/quickstart.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Quickstart | Canonical Distribution of Kubernetes&reg;"
+keywords: quickstart
+tags: [getting_started]
+sidebar: k8smain-sidebar
+permalink: quickstart.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+<style>
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 75%;
+  height: auto;
+}
+</style>
+
+<div class="p-content__row">
+    <div class="u-equal-height">
+        <div class="col-6">
+
+<h2> Quick Start</h2>
+
+<p>The Canonical Distribution of Kubernetes<sup>&reg;</sup> delivers a ‘pure K8s’
+experience, tested across a wide range of clouds and integrated with modern
+metrics and monitoring. It works across all major public clouds and private
+infrastructure, enabling your teams to operate Kubernetes clusters on demand,
+anywhere.</p>
+
+<p>With this quick start guide and some tools from Canonical, you can have all
+this distilled Kubernetes expertise operating in your own  cluster
+running on a cloud of your choice in minutes!</p>
+
+</div>
+<div class="col-6 u-align--right">
+    <img style="border: 0" src="./images/k8s-bundle.png" width="350" height="350">
+</div>
+</div></div>
+
+<hr class="is-deep">
+
+## What you'll need
+
+  *  An Ubuntu 18.04 LTS or 16.04 LTS environment to run the commands (or another operating system which supports `snapd` - see the [snapd documentation][snapd-docs])
+  * Account credentials for one of the following public clouds:
+      -  [Amazon Web Services][cloud-aws], including AWS-China and AWS-gov
+      -  [CloudSigma][cloud-cloudsigma]
+      - [ Google Cloud platform ][cloud-google]
+      - [Joyent][cloud-joyent]
+      - [Microsoft Azure][cloud-azure], including Azure-China
+      - [Oracle Cloud][cloud-oracle]
+      - [Rackspace][cloud-rackspace]
+
```

Comment:
This list makes me cringe since we're only testing on AWS, Google, and Azure right now. But I guess we should leave all these options since conjure-up lists them.

Reply:
I got it to work on Oracle once, so... :expressionless: